src/main/java/pl/tomaszdziurko/jvm_bloggers/view/admin/mailing/MailingPageRequestHandler.java

```diff
@@ -27,12 +27,12 @@ public MailingPageRequestHandler() {
     @Autowired
     public MailingPageRequestHandler(BlogSummaryMailGenerator blogSummaryMailGenerator,
```

Comment:
You can remove hand-crafted constructor and replace it with `@RequiredArgsConstructor(onConstructor = @__(@Autowired))` class level annotation. See the [example](https://github.com/tdziurko/jvm-bloggers/blob/master/src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/AggregatedRssFeedProducer.java#L32).